soh/src/code/game.c

```diff
@@ -458,8 +458,7 @@ void GameState_Update(GameState* gameState) {
         }
     }
 
-    //since our CVar is same value and properly default to 0 there is not problems doing this in single line.
```

Comment:
i do not really understand this comment we are removing here, should I be concerned with it?

Reply:
i think it's just explaining that we get default values from cvars, which is a common pattern throughout the codebase so i figured the comment wasn't necessary